tox.ini

```diff
@@ -0,0 +1,31 @@
+[tox]
+envlist = py27
+
+
+[testenv]
+basepython = python2.7
+deps =
+    coverage
+    flake8
+    flake8_docstrings
+    mock
+    pytest
+    pytest-capturelog
+    Flask==0.10.1
+    Jinja2==2.7.3
+    MarkupSafe==0.23
+    MySQL-python==1.2.5
+    Werkzeug==0.9.6
+    argparse==1.2.1
+    itsdangerous==0.24
+    requests==2.9.1
+    wsgiref==0.1.2
```

Comment:
do we need all of these dependencies to make this run?  it seems like a lot.  I don't understand mysql-python, flask, jinja2, requests, wsgiref.  If that is required for running tox, lets add a comment if we can.

Also can we add a comment below to the [flake8] section where ignore is- I would like to know what those magic codes mean (such as D100, E731)